1-foundations/1-types-and-syntax/foundations-s1.ipynb

```diff
@@ -808,12 +973,130 @@
    "outputs": [],
    "source": [
     "# Define a string and an int\n",
-    "a_str = \"three\"\n",
+    "a_str = 'three'\n",
     "a_int = 3\n",
     "\n",
     "# Print variables and the result the variable is an int or not\n",
-    "print('Variable a_str:', a_str, \"is an int:\", isinstance(a_str, int))\n",
-    "print('Variable a_int:', a_int, \"is an int:\", isinstance(a_int, int))"
+    "print('Variable a_str:', a_str, 'is an int:', isinstance(a_str, int))\n",
+    "print('Variable a_int:', a_int, 'is an int:', isinstance(a_int, int))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Defining your own booleans:**\n",
+    "\n",
+    "You can define your own boolean in two ways:\n",
+    "\n",
+    "* setting a variable to `True` or `False`\n",
+    "* setting a variable to the result of a logical expression. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "my_true_boolean = True\n",
+    "my_false_boolean = False\n",
+    "\n",
+    "print('Variable my_true_boolean:', my_true_boolean, type(my_true_boolean))\n",
+    "print('Variable my_false_boolean:', my_false_boolean, type(my_false_boolean))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Logical expressions are equations that evaluates to true or false. The most common logical expressions use the `==`, `and` or `or` operators.\n",
```

Comment:
I suggest adding `!=`

Reply:
Done in 014a631025bdfb36f751e32a01feeb0f4758ee38 with some more stuff